library/haproxy

```diff
@@ -14,24 +14,24 @@ Architectures: amd64, arm32v6, arm32v7, arm64v8, i386, ppc64le, s390x
 GitCommit: d87e27dd90273fe213f7d468516ad192e959f086
 Directory: 2.5-rc/alpine
 
-Tags: 2.4.1, 2.4, lts, latest
+Tags: 2.4.2, 2.4, lts, latest
 Architectures: amd64, arm32v5, arm32v7, arm64v8, i386, mips64le, ppc64le, s390x
-GitCommit: 8cac7162fb3fb4431208d98c2cd9c63d1897421f
+GitCommit: 973f43b2d1c5c0d57274002e216761c0baedf6c0
 Directory: 2.4
 
-Tags: 2.4.1-alpine, 2.4-alpine, lts-alpine, alpine
+Tags: 2.4.2-alpine, 2.4-alpine, lts-alpine, alpine
 Architectures: amd64, arm32v6, arm32v7, arm64v8, i386, ppc64le, s390x
-GitCommit: 8cac7162fb3fb4431208d98c2cd9c63d1897421f
+GitCommit: 973f43b2d1c5c0d57274002e216761c0baedf6c0
 Directory: 2.4/alpine
 
-Tags: 2.3.10, 2.3
+Tags: 2.3.11, 2.3
```

Comment:
2.3.11 is unsafe, do not merge before including 2.3.12: https://www.mail-archive.com/haproxy@formilux.org/msg40894.html